docs/sources/panels/transform-data/index.md

```diff
@@ -568,3 +643,32 @@ Here is the result after adding a Limit transformation with a value of '3':
 | 2020-07-07 11:34:20 | Temperature | 25    |
 | 2020-07-07 11:34:20 | Humidity    | 22    |
 | 2020-07-07 10:32:20 | Humidity    | 29    |
+
+### Join by field (Inner join)
```

Comment:
This does not exist in 9.1... so should be removed from the backport

Reply:
removed.